.travis.yml

```
@@ -18,6 +18,14 @@ test-extensions-job-template: &test-extensions-job-template
 
 jobs:
   include:
+    - stage: Pre-test checks
+      name: Check coding style
+      if: type = pull_request
+      language: go
+      go: '1.13'
+      before_script: GO111MODULE=on go install mvdan.cc/sh/v3/cmd/shfmt
```

```suggestion
      before_script: go get github.com/mvdan/sh/cmd/shfmt
```

since go1.13 is used, i would remove GO111MODULE

---

:+1: see db53c3585ca6ee48f78893d3466b8378f3023624

---

:+1: see db53c3585ca6ee48f78893d3466b8378f3023624

---

It seems that `GO111MODULE` is required: see https://travis-ci.org/mlocati/docker-php-extension-installer/jobs/627782173#L227